worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +53,75 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// Initialise variables
+	std::string guess;
+	int maxLives = 4;
```

Comment:
Could make this a `const` since it shouldn't change over the course of the program